src/App.js

```diff
@@ -1,11 +1,80 @@
+import TodoList from './app/components/TodoList';
+import SignUp from './app/components/forms/SignUp';
+import LogIn from './app/components/forms/LogIn';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+  Redirect
+} from "react-router-dom";
+
+import UserApi from './app/services/UserApi';
 var React = require('react');
 var ReactDOM = require('react-dom');
+
 export default class App extends React.Component {
```

Comment:
Using functional components is considered a more modern approach, you can use hooks to implement state and lifecycle hooks there. Class components are generally more clumped up. Functional components are faster and easier to maintain.
https://reactjs.org/docs/components-and-props.html#function-and-class-components